src/lib/homestore_backend/hs_homeobject.cpp

```diff
@@ -237,41 +237,6 @@ void HSHomeObject::init_homestore() {
             });
         }
 
-        // We dont have any repl dev now, explicitly call init_completed_cb() where we register PG/Shard meta types.
-        repl_app->on_repl_devs_init_completed();
```

Comment:
this will be called when initializing raft_repl_service, no need to call it here explicitly